orchagent/aclorch.cpp

```diff
@@ -4741,18 +5465,18 @@ void AclOrch::doAclRuleTask(Consumer &consumer)
             }
 
             if (bHasIPV4 && bHasIPV6)
-	    {
-		    if (type == TABLE_TYPE_L3V4V6)
-		    {
-			    SWSS_LOG_ERROR("Rule '%s' is invalid since it has both v4 and v6 matchfields.", rule_id.c_str());
-			    bAllAttributesOk = false;
-		    }
-	    }
+	        {
+		        if (type == TABLE_TYPE_L3V4V6)
+		        {
+			        SWSS_LOG_ERROR("Rule '%s' is invalid since it has both v4 and v6 matchfields.", rule_id.c_str());
+			        bAllAttributesOk = false;
+		        }
+	        }
 
             // validate and create ACL rule
             if (bAllAttributesOk && newRule->validate())
             {
-                if (addAclRule(newRule, table_id))
+                if (addAclRuleWithEgrSetDscp(newRule, table_id))
```

Comment:
Same here. Suggest keeping the original function name

Reply:
again, I would prefer that the reader can clearly distingush that addAclRuleWithEgrSetDscp handles a special case and addAclRule is a general rule handler.